charts/sn-platform/templates/streamnative-console/streamnative-console-service.yaml

```diff
@@ -41,7 +41,11 @@ spec:
     - name: frontend
       port: {{ .Values.streamnative_console.service.ports.frontend }}
       protocol: TCP
-      targetPort: {{ .Values.streamnative_console.ports.frontend }}
+      targetPort: 9527
```

Comment:
Could `targetPort` use predefined values as well?

Reply:
I currently recommend not using this predefined value, because currently the backend service starts on port 7750 and the frontend service starts on port 9527, the backend service port is configured in the configuration file and the frontend port is configured in nginx, if we change it, we need to do some work, if we configure it with the predefined value but the port is not really used in the service, it may cause the request not to be forwarded to the service, so I'll create an issue, and we'll implement it later